### PR TITLE
Add host level auto update setting with default true

### PR DESCRIPTION
**Pull request by alphadose**
## Describe your changes
Add host level auto update setting with default true

## Provide Issue ticket number if applicable/not in title
https://toil.kitemaker.co/cQARgD-Gravitl/Nk7Fgo-Main_Board/items/1374

## Checklist before requesting a review
- [x] My changes affect only 10 files or less.
- [ ] I have performed a self-review of my code and tested it.
- [x] If it is a new feature, I have added thorough tests, my code is <= 1450 lines.
- [ ] If it is a bugfix, my code is <= 200 lines.
- [x] My functions are <= 80 lines.
- [ ] I have had my code reviewed by a peer.
- [ ] My unit tests pass locally.
- [x] Netmaker is awesome.

### CODE REVIEW

**Review by Aceix — APPROVED** [review]
Looks good but "NETCLIENT_AUTO_UPDATE" sounds more specific

**Review by 0xdcarns — CHANGES_REQUESTED** [review]

**Review by mattkasun — APPROVED** [review]

File: servercfg/serverconf.go [hunk 1]
```diff
@@ -382,6 +382,12 @@ func GetVerbosity() int32 {
 	return int32(verbosity)
 }
 
+// AutoUpdateEnabled returns a boolean indicating whether netclient auto update is enabled or disabled
+// default is enabled
+func AutoUpdateEnabled() bool {
+	return os.Getenv("NETCLIENT_AUTO_UPDATE") != "disabled"
```
[the remark this answers is not in the document]
Comment (alphadose) [reply servercfg/serverconf.go]: @abhishek9686 done

**Comment by theguy951357** [comment]
I need testing steps for this one please

**Review by theguy951357 — APPROVED** [review]
pr steps pass
critical list passes

**Pull request merged**